src/vmware/azext_vmware/tests/latest/recordings/test_vmware.yaml

```diff
@@ -296,11 +288,7 @@ interactions:
       ParameterSetName:
       - -g
       User-Agent:
-      - python/3.6.9 (Linux-4.19.128-microsoft-standard-x86_64-with-Ubuntu-18.04-bionic)
-        msrest/0.6.13 msrest_azure/0.6.3 avsclient/2020-03-20 Azure-SDK-For-Python
-        AZURECLI/2.7.0
-      accept-language:
-      - en-US
+      - AZURECLI/2.18.0 azsdk-python-avsclient/unknown Python/3.8.7 (Windows-10-10.0.19041-SP0)
```

Comment:
Is `azsdk-python-avsclient/unknown` okay here?

Reply:
If test can pass, then no problem.